static/elements/chromedash-feature.js

```diff
@@ -0,0 +1,492 @@
+import {LitElement, html} from 'https://unpkg.com/@polymer/lit-element@latest/lit-element.js?module';
+import 'https://unpkg.com/@polymer/iron-icon/iron-icon.js?module';
+import '/static/elements/chromedash-color-status.js';
+
+const MAX_STANDARDS_VAL = 6;
+const MAX_VENDOR_VIEW = 7;
+const MAX_WEBDEV_VIEW = 6;
+const MAX_RISK = MAX_VENDOR_VIEW + MAX_WEBDEV_VIEW + MAX_STANDARDS_VAL;
+
+const IS_PUSH_NOTIFIER_ENABLED = window.PushNotifier.GRANTED_ACCESS;
+const IS_PUSH_NOTIFIER_SUPPORTED =
+    window.PushNotifier && window.PushNotifier.SUPPORTS_NOTIFICATIONS;
+
+class ChromedashFeature extends LitElement {
+  static get properties() {
+    return {
+      feature: {type: Object}, // From attribute
+      whitelisted: {type: Boolean}, // From attribute
+      open: {type: Boolean, reflect: true}, // Attribute used in the parent for styling
+      // Values used in the template
+      _interopRisk: {type: Number, attribute: false},
+      _isDeprecated: {type: Boolean, attribute: false},
+      _hasDocLinks: {type: Boolean, attribute: false},
+      _hasSampleLinks: {type: Boolean, attribute: false},
+      _commentHtml: {type: String, attribute: false},
+      _crBugNumber: {type: String, attribute: false},
+      _newBugUrl: {type: String, attribute: false},
+      _receivePush: {type: Boolean, attribute: false},
+    };
+  }
+
+  constructor() {
+    super();
+    this.open = false;
+  }
+
+  // Initialize values after receiving `this.feature`.
+  firstUpdated() {
+    this._receivePush = this.feature.receivePush;
+  }
+
+  get _crBugNumber() {
+    const link = this.feature.browsers.chrome.bug;
+    if (!link) {
+      return '';
+    }
+
+    /* Get the number id from a url.
+     * Url has two formats: "http://crbug.com/111111", and
+     * "https://bugs.chromium.org/p/chromium/issues/detail?id=111111" */
+    const matches = link.match(/(id=|crbug.com\/)([0-9]+)/);
+    if (matches) {
+      return matches[2];
+    }
+    return '';
+  }
+
+  set _crBugNumber(value) {/* Never called */}
```

Comment:
If it's never called, you can omit the setter. Then a accidental call will cause an error.

Reply:
If I don't have this line, the getter won't work.

reduced test case: This component renders text "foo". But when I comment out the setter, it does not. Is it a bug in lit-element?

class MyGetter extends LitElement {
  static get properties() {
    return {
      test: {type: String},
    };
  }

  get test() {
    return 'foo';
  }

  // set test(foo) {
  //   return;
  // }

  render() {
    return html`${this.test}`;
  }
}

customElements.define('my-getter', MyGetter);

Reply:
Never mind. I updated them all to one time assignment instead of getters. (since the values do not change).

Regarding the getter/setter issue on lit-element: After I upgrade lit-element from rc to latest, it works as expected now.